third_party/libusb/naclport/src/libusb_over_chrome_usb_unittest.cc

```diff
@@ -435,7 +435,7 @@ class LibusbOverChromeUsbTransfersTest
                 Call(GetExpectedTransferStatus(transfer_index, is_canceled)));
   }
 
-  void SetUpTransferCallbackMockExpectations(
+  static void SetUpTransferCallbackMockExpectations(
       size_t transfer_index,
       bool /*is_output*/,
```

Comment:
could we remove this parameter since it's not used anyway?

Reply:
Indeed! Done.
(I guess I added them initially in order to follow the interface of other similar methods, but that's probably not very useful.)